components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -1110,22 +1102,13 @@ private String getNextStringInLexicalOrder(String str) {
         return new String(bytes);
     }
 
-    private OMElement queryDatabase(String columnFamily,
-                                    QueryServiceStub.CompositeIndex[] compositeIndex)
-            throws APIMgtUsageQueryServiceClientException {
+    private OMElement queryDatabase(String columnFamily) throws APIMgtUsageQueryServiceClientException {
```

Comment:
Wrong answer :). If you touch the code, you are the owner. More than that, if you feel the code could be improved, you should always improve it, regardless of who wrote the code. It is bad to say "I didn't write this" :).